app/common/runtimeTypes/storage/index.ts

```diff
@@ -0,0 +1,10 @@
+import * as t from "io-ts"
+
+export const CURRENT_WALLETS_VERSION = 0
+export const CURRENT_WALLET_VERSION = 0
+export const CURRENT_TRANSACTIONS_VERSION = 0
+
+export const Version = t.type({
+  _v: t.number
+}, "Version")
+export type Version = t.TypeOf<typeof Version>
```

Comment:
I'm not sure whether these changes are to be implemented anymore... When talking about the implementation of #401, we decided that transactions are not to be saved in storage for the time being. Keeping that in mind, I think we don't need to version transactions at this point and so, I would remove the definition of CURRENT_TRANSACTIONS_VERSION.

Reply:
Done :stuck_out_tongue_winking_eye: